src/core/attribute.d

```diff
@@ -136,8 +136,14 @@ version (UdaGNUAbiTag) struct gnuAbiTag
 {
     string[] tags;
 
-    this(string[] tags...)
+    this(scope string[] tags)
     {
         this.tags = tags;
     }
+
+    /* Overloads for backwards compatibility
+     */
+    this(string s1)                       { this.tags = [s1]; }
```

Comment:
This is a rabbit hole. Maybe it's better moving the variadic part from the parameters to the template definition.